msi/GitProduct.wxs

```diff
@@ -284,6 +286,9 @@
       <Custom Action="ExecConfigureCoreAutoCrlf" After="ExecPostInstallBat">
         <![CDATA[REMOVE <> "ALL"]]>
       </Custom>
+      <Custom Action="ExecConfigureSslCAInfo" After="ExecConfigureCoreAutoCrlf">
+        <![CDATA[REMOVE <> "ALL"]]>
+      </Custom>
```

Comment:
I cannot see the full context in GitHub's new and improved PR handling, but I seem to recall that this diff hunk was pretty lonely in the commit and wanted to join its friends in another commit. IOW I think that hunk simply was not covered by the commit message, but would have fit perfectly with another commit's message.